teamwork/apps/profiles/views/EditProfileView.py

```diff
@@ -157,19 +175,18 @@ def edit_profile_helper(request, username):
         Helper function that saves profile information from the ProfileForm
     """
 
```

Comment:
Please revert the below change. Add a `tempProfile` to the else: instead of pulling outside.